remappings.txt

```diff
@@ -1,5 +1,6 @@
 solmate/=lib/solmate/
 permit2/=lib/permit2/
 forge-std/=lib/forge-std/src/
-@openzeppelin/=node_modules/@openzeppelin
+@openzeppelin/contracts/token/ERC721/=lib/ERC721
+@openzeppelin/contracts-v4=node_modules/@openzeppelin/contracts-v4
```

Comment:
what exactly happened to lead to this setup with ERC721 logic inside a different folder?